evidence/pool.go

```diff
@@ -96,34 +98,54 @@ func (evpool *Pool) Update(block *types.Block, state sm.State) {
 
 // AddEvidence checks the evidence is valid and adds it to the pool.
 func (evpool *Pool) AddEvidence(evidence types.Evidence) error {
-
-	// check if evidence is already stored
-	if evpool.store.Has(evidence) {
-		return ErrEvidenceAlreadyStored{}
+	state := evpool.State()
+	valSet, err := sm.LoadValidators(evpool.stateDB, evidence.Height())
+	if err != nil {
+		return errors.Wrapf(err, "can't load validators at height #%d", evidence.Height())
```

Comment:
```suggestion
		return fmt.Errorf("can't load validators at height #%d, %w", evidence.Height(), err)
```

to keep inline with the styleguide